### PR TITLE
Ades/aave/create calculation in usdc instead of eth

### PR DESCRIPTION
**Pull request by sennett**
# [Create calculation in USDC instead of ETH
](https://app.shortcut.com/oazo-apps/story/6424/calculations-create-calculation-in-usdc-instead-of-eth)

- Formats things in order info panel to USDC instead of ETH. 
- Provides an observable that can be reused.
  
## How to test 🧪
- Create AAVE earn position - should be no regression.
- Go to create AAVE multiply position.  It should show in USDC.

### CODE REVIEW

File: components/AppContext.ts [hunk 1]
```diff
@@ -841,7 +842,7 @@ export function setupAppContext() {
 
   const aaveUserReserveData$ = observe(onEveryBlock$, context$, getAaveUserReserveData)
   const aaveUserAccountData$ = observe(onEveryBlock$, context$, getAaveUserAccountData)
-  const aaveOracleAssetPriceData$ = observe(onEveryBlock$, context$, getAaveOracleAssetPriceData)
+
```
Comment (jakubswierczek) [inline components/AppContext.ts:845]: too much oxygen here

**Review by jakubswierczek — CHANGES_REQUESTED** [review]

File: features/aave/common/components/informationContainer/StrategyInformationContainer.tsx [hunk 2]
```diff
@@ -48,6 +48,7 @@ export function StrategyInformationContainer({ state }: OpenAaveInformationConta
         {...state.context}
         transactionParameters={strategy}
         currentPosition={currentPosition}
+        debtToken={state.context.token}
```
[the remark this answers is not in the document]
Comment (sennett) [reply features/aave/common/components/informationContainer/StrategyInformationContainer.tsx:51]: This has changed in the other PR with the generic flow.

File: features/aave/common/components/informationContainer/FeesInformation.tsx [hunk 3]
```diff
@@ -25,9 +27,16 @@ export function FeesInformation({
...
   const { t } = useTranslation()
   const [showBreakdown, setShowBreakdown] = React.useState(false)
-  const fee = transactionParameters.simulation.swap.targetTokenFee.plus(
+  const swapFee = transactionParameters.simulation.swap.targetTokenFee.plus(
     transactionParameters.simulation.swap.sourceTokenFee,
   )
+
+  const { convertToAaveOracleAssetPrice$ } = useAppContext()
+
+  const [currentDebtInDebtToken] = useObservable(
```
[the remark this answers is not in the document]
Comment (sennett) [reply features/aave/common/components/informationContainer/FeesInformation.tsx:36]: Good spot. FIxed.

File: features/aave/common/components/informationContainer/OutstandingDebtInformation.tsx [hunk 4]
```diff
@@ -3,33 +3,48 @@ import { Flex } from '@theme-ui/components'
...
 
 export function OutstandingDebtInformation({
   transactionParameters,
   currentPosition,
+  debtToken,
 }: OutstandingDebtInformationProps) {
   const { t } = useTranslation()
 
-  const { amount, denomination = 'ETH' } = transactionParameters.simulation.position.debt
+  const { convertToAaveOracleAssetPrice$ } = useAppContext()
```
[the remark this answers is not in the document]
Comment (sennett) [reply features/aave/common/components/informationContainer/OutstandingDebtInformation.tsx:27]: We can use swap price.  I will update in next PR.

**Review by jakubswierczek — APPROVED** [review]

**Pull request merged**